src/Subscriptions/Subscriber.php

```diff
@@ -119,7 +119,11 @@ public function __unserialize(array $data): void
     /** Generate a unique private channel name. */
     public static function uniqueChannelName(): string
     {
-        return 'private-lighthouse-' . Str::random(32) . '-' . time();
+        $channelType = config('lighthouse.subscriptions.encrypted_channels', false)
+            ? 'private-encrypted'
+            : 'private';
+
+        return $channelType . '-lighthouse-' . Str::random(32) . '-' . time();
```

Comment:
Is 32 still appropriate when the prefix is longer?

Reply:
Well, the random is based on random_bytes, so should be fine. Anyway there's also the time.